src/view/components/toolbar/motion/Accelerometer.tsx

```diff
@@ -44,12 +46,27 @@ interface IProps {
         Z_AXIS: number;
     };
     onUpdateValue: (sensor: SENSOR_LIST, value: number) => void;
+    onSelectGestures?: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+    onSendGesture?: () => void;
 }
+
+const GESTURE_BUTTON_MESSAGE = "Send Gesture";
+
 export const Accelerometer: React.FC<IProps> = (props: IProps) => {
     return (
         <div className="AccelerometerBar">
             <br />
             {/* Implement Gestures Here */}
+            <Dropdown options={GESTURES} onSelect={props.onSelectGestures} />
+            <SensorButton
+                label={GESTURE_BUTTON_MESSAGE}
+                onMouseDown={() => {
+                    if (props.onSendGesture) {
+                        props.onSendGesture();
+                    }
+                }}

```

Comment:
can we have an onMouseUp event that sends `{"gesture": ""}`?
This is because most of the time a micro:bit is not in a gesture.

Reply:
Should I send this {"shake":true} on mousedown then {"shake":false} on mouse up

Reply:
or do you prefer empty string

Reply:
I think empty string is better

Reply:
Got it